pipeline/08-api.R

```diff
@@ -39,17 +37,10 @@ towns <- ccao::town_dict %>%
 dict <- ccao::vars_dict %>%
   filter(
     var_data_type == "categorical",
-    var_name_model %in% predictors,
-    var_name_model != "meta_modeling_group"
```

Comment:
`meta_modeling_group` is no longer used as a predictor, so I've removed its special handling from this code block and the block that follows.